ci/azure-pipeline-nightly.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: $(SourceBranchName)-$(Date:yyyyMMdd)$(Rev:.rrr)
+
+trigger: none
+pr: none
+schedules:
+  - cron: "0 6 * * *"
+    displayName: Daily midnight build
```

Comment:
Specify at what time (UTC/EST) these builds will trigger. helps for quick check. Seems this triggers at morning 6:00 AM EST.

Reply:
This triggers at 12:00AM EST. I will adjust it to an hour before daily PTE runs

Reply:
And add a comment as to what time it triggers